lib/browser/ipc-main-internal-utils.ts

```diff
@@ -15,15 +15,11 @@ const callHandler = async function (handler: IPCHandler, event: ElectronInternal
 export const handle = function <T extends IPCHandler> (channel: string, handler: T) {
   ipcMainInternal.on(channel, (event, requestId, ...args) => {
     callHandler(handler, event, args, responseArgs => {
-      event._replyInternal(`${channel}_RESPONSE_${requestId}`, ...responseArgs)
-    })
-  })
-}
-
-export const handleSync = function <T extends IPCHandler> (channel: string, handler: T) {
-  ipcMainInternal.on(channel, (event, ...args) => {
-    callHandler(handler, event, args, responseArgs => {
-      event.returnValue = responseArgs
+      if (requestId) {
```

Comment:
How do check if it's really the `requestId` and not one of the `args`?

Reply:
why would it be one of the args? `invokeSync` sends a `null` there